blocks/alert-bar-block/features/alert-bar/default.jsx

```diff
@@ -141,15 +143,20 @@ class AlertBar extends Component {
       return null;
     }
 
-    const { arcSite } = this.props;
+    const { arcSite, customFields } = this.props;
+    const { ariaLabel } = customFields;
     const { content_elements: elements = [] } = content;
     const article = elements[0] || {};
     const { websites = {}, headlines = {} } = article;
     const { website_url: websiteURL = '' } = websites[arcSite] || {};
 
     return (
       !!content?.content_elements?.length && visible && (
-        <nav className="alert-bar" ref={this.alertRef}>
+        <nav
+          className="alert-bar"
+          ref={this.alertRef}
+          aria-label={ariaLabel || 'Breaking News Alert'}
```

Comment:
You have a default value here as well as in the `defaultProps` we can remove one in favour of another.

Reply:
I left this one here and removed the defaultProps